buildout.cfg

```diff
@@ -1,33 +1,8 @@
 [buildout]
-extends =
-    https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg
-    https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
-
+extends = https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
```

Comment:
Can you guys please explain this to me?  I have no idea how this travis stuff works.  Why do 4.3 tests use Plone 5?